doc/tech/README.md

```diff
@@ -3,203 +3,170 @@
-- **Reference Implementation(RI)**: Builds on the requirements and specifications developed in RM, RAs and adds details so that it can be implemented.  Each Reference Architecture is expected to be implemented by at least one Reference Implementation.
-- **Reference Certification(RC)**: Builds on the requirements and specifications developed in the other documents and adds details on how an implementation will be verified, tested and certified. Both infrastructure verification and certification as well as VNFs/CNFs verifications and certifications will be covered.
-
-Below is a diagram of the different artifacts that will need to be created to support the implementation of the abstract concepts presented in the **Reference Model**, which are then applied to create the **Reference Architecture**, that will be deployed using the requirements spelled out in the **Reference Implementation**.
-
-<p align="center"><img src="./figures/tech_scope_3.png" alt="scope" title="Document Types" width="100%"/></p>
-<p align="center"><b>Figure 2:</b> Description of the possible different levels of CNTT artefacts</p>
-
-<a name="3.0"></a>
-## 3. Principles
-
-Any specification work created within CNTT **must** conform to set of principles specified by CNTT:
-
-* [Reference Model Principles](../ref_model/chapters/chapter01.md#13-principles)
-* [Reference Architecture Principles](../ref_arch#principles)
-
-<a name="4.0"></a>
-## 4. Relevant Technologies
-This section describes the relevant technologies for CNTT and clarifies CNTT position about them. The list below is not exhaustive but it is representative of existing trends. Chapter 5 describe CNTT policies of managing the Non-Conferment ones. 
-
-<a name="4.1"></a>
-### 4.1 IO Virtualization
-
-There are different ways of which IO devices (such as NICs) are presented to workloads for consumption by those workloads. Here is a list of current methods of existing IO Virtualization:
-
-- Para-Virtualization method (software only).
-- Direct Assignment via h/w assisted PCI-Passthrough (IOMMU).
-- Device Sharing with SR-IOV & h/w assisted PCI-Passthrough (IOMMU).
-- Para-Virtualization method with Hardware support.
+<a name="1.1"></a>
+## 1.1 Glossary
+The definition and intent of the terminology used throughout the documents is defined in the [Glossary](./glossary.md).
 
-**Figure 3** below shows some of the relevant IO Virtualization techniques.
+<a name="1.2"></a>
+## 1.2 Problem Statement
+Based on informal conversations with many operators and developers, there is a realization that there are significant technical, operational and business challenges to the development and deployment of VNF applications related to the lack of a common virtualized infrastructure platform.  These include but are not limited to the following:
```

Comment:
realization -> realisation

Reply:
fixed in: https://github.com/cntt-n/CNTT/pull/1267/commits/be795526541d9536a966dbb42e940cfd45b05852